### DISCUSSION TITLE
[REFLET] Reflet de la loi d'eau

### DISCUSSION THREAD

**Discussion by ABIAdrienG** [witsa/synapps#573] [Q&A]
Bonjour,

Je souhaiterais utiliser le reflet de la loi d'eau pour l'afficher sur Synapps cependant, je le trouve peu pratique.

La taille du graphique est toute petite comparée à la taille des zones de saisies. 
Les deux éléments se dimensionnent avec la taille du texte, ils ont donc un ratio constant.

J'imaginais l'affichage du graphique lors de la visualisation de la frame et le remplacement de celui-ci par les zones de saisies lors de la pression d'un bouton réglage. (on peut imaginer l'affichage de la loi d'eau en transparence lors de la saisie).

Je n'ai malheureusement pas encore assez de connaissances en Js pour faire un composite de toutes pièces.

Est-ce que des modifications de ce reflet sont prévues ?

Merci par avance pour votre réponse.

**Comment by FlorentAdell** [comment]
Bonjour, 

Pour le reflet courbe de chauffe (Loi d'eau), j'augemente la taille du texte

![image](https://user-images.githubusercontent.com/46195964/226375629-3711f2b8-20c8-41e8-8794-57420dbcb047.png)

![image](https://user-images.githubusercontent.com/46195964/226375856-4777d70e-b42b-40ef-bb25-eb4923e114dc.png)

Je n'ai pas encore essayé de refaire un composite courbe de chauffe

**Reply by ABIAdrienG** [reply]
Merci pour votre réponse.
J'utilise aussi la taille de texte.
Cependant, ça ne règle pas le problème de ratio entre le graphique et les zones de saisies.

J'ai été obligé de rester sur l'ancien système avec une frame sur synapps.
C'est dommage car la nouvelle est bien plus belle mais elle n'est pas adaptée à mes projets.

**Reply by FlorentAdell** [reply]
C'est vrai, que pouvoir modifier les points sur la courbe directement avec la souris, c'est dommage de ne pas l'avoir sur Synapps comme sur la courbe de chauffe dans le Wit directement
